CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * `jj backout --revision` has been renamed to `jj backout --revisions`.
   The short alias `-r` is still supported.
 
+* [The default `immutable_heads()` set](docs/config.md#set-of-immutable-commits)
```

Comment:
Assuming this is true, you should mention that untracked branches no longer show up in the `jj log` default revset.

Reply:
Update the changelog entry.